doc/release-notes.md

```diff
@@ -1,3 +1,37 @@
+# Release 0.14.1
+## Theme
+Ziti 0.14.1 includes the following:
+
+### Features
+Ability to set precedence/cost after Listen. This enables graceful shutdown or client side adaptive costing
+
+### Fixes
+  * Fixed race condition in end-to-end encryption setup
+  * Xt fixes
+      * Fixed strategies missing session ended events
```

Comment:
any chance we can link the issue id to these?

Reply:
I didn't put any issues in for these. Longer term we'll may want to track at that granular level, but I don't know if we're there yet